cli/command/image/build/context_test.go

```diff
@@ -129,7 +129,6 @@ func TestGetContextFromReaderString(t *testing.T) {
 	tarReader := tar.NewReader(tarArchive)
 
 	_, err = tarReader.Next()
-
```

Comment:
nit: Accidental line removal?

Reply:
No, looks like gofumpt tightened up formatting a bit for these so that there's no empty line between where the error is produced and where it's checked.

Reply:
Looks like it's this commit; https://github.com/mvdan/gofumpt/commit/025a91f64d518cb7e1c399428a61685d513c4214


previously it would already do this for the `err := ...` case, but now also for `err = ...` (pre-existing `err` variable) case.